src/program/lwaftr/run/README

```diff
@@ -1,15 +1,41 @@
 Usage: run --help
-       run --conf <conf-file> --v4 <pci-addr> --v4 <pci-addr> [OPTION...]
+       run --conf <conf-file> --v4 [<kind>:]<config> --v6 [<kind>:]<config> [OPTION...]
 
 Required arguments:
-       --conf   <conf-file>     Sets configuration policy table
-       --v4 <pci-addr>          PCI device number for the INET-side NIC
-       --v6 <pci-addr>          PCI device number for the B4-side NIC
+       --conf <conf-file>         Sets configuration policy table.
+       --v4 [<kind>:]<config>     Device kind and configuration parameters
+                                  for the Internet-side interface.
+       --v6 [<kind>:]<config>     Device kind and configuration parameters
+                                  for the B4-side interface.
 
 Optional arguments:
-       --virtio                 Use virtio-net interfaces instead of Intel 82599
-       --ring-buffer-size <size>  Set Intel 82599 receive buffer size
-       --cpu    <cpu>           Bind the lwAFTR to the given CPU
-       --real-time              Enable real-time SCHED_FIFO scheduler
-       -D       <seconds>       Duration in seconds
-       -v                       Verbose (repeat for more verbosity)
+       --ring-buffer-size <size>  Set the Intel 82599 receive buffer size.
+       --cpu <cpu>                Bind the lwAFTR to the given CPU.
+       --real-time                Enable real-time SCHED_FIFO scheduler.
+       -D <seconds>               Duration in seconds.
+       -v                         Verbose (repeat for more verbosity).
+
+Deprecated arguments:
+       --v4-pci <pci-addr>        PCI device number for the INET-side NIC.
+       --v6-pci <pci-addr>        PCI device number for the B4-side NIC.
+       --virtio                   Use virtio-net interfaces instead of Intel 82599.
+
+Device kinds and configuration parameters:
+
+       intel82259:<pci-addr>
```

Comment:
82599, not 82259, here and below.

With PCI addresses, is it necessary to specify the kind?  AFAIU we should be able to get this information from /sys/.  See first snippet in https://github.com/lukego/blog/issues/13.